lib/dbservice/users/student.ex

```diff
@@ -15,13 +16,32 @@ defmodule Dbservice.Users.Student do
     field :mother_phone, :string
     field :stream, :string
     field :uuid, :string
+    field :physically_handicapped, :boolean
+    field :family_income, :string
+    field :father_profession, :string
+    field :father_education_level, :string
+    field :mother_profession, :string
+    field :mother_education_level, :string
+    field :time_of_device_availability, :date
+    field :has_internet_access, :boolean
+    field :primary_smartphone_owner, :string
+    field :primary_smartphone_owner_profession, :string
     belongs_to :user, User
     belongs_to :group, Group
+    has_many :enrollment_record, EnrollmentRecord
 
     timestamps()
   end
 
   @doc false
+  @spec changeset(
+          {map, map}
+          | %{
+              :__struct__ => atom | %{:__changeset__ => map, optional(any) => any},
+              optional(atom) => any
+            },
+          :invalid | %{optional(:__struct__) => none, optional(atom | binary) => any}
+        ) :: Ecto.Changeset.t()
```

Comment:
I think we had not been adding spec for changesets. That's why `@doc false` was there I believe. Can you share how this is showing up on swagger?